<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>326</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_exp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0)</string>
          </property>
          <property name="text">
           <string>Выражение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_exp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0);</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_x">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0)</string>
          </property>
          <property name="text">
           <string>x = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_x">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0);</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0);</string>
          </property>
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear_one_sym">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(102, 255, 0);</string>
          </property>
          <property name="text">
           <string>⌫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_sin">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>sin(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tan">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0)</string>
            </property>
            <property name="text">
             <string>tan(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_asin">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>asin(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_sqrt">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>sqrt(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_log">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>log(</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_cos">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>cos(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_acos">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>acos(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_atan">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>atan(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ln">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>ln(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_pow">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>^</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_openb">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_1">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0)</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_0">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_closeb">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_x">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_mod">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_point">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_div">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_mult">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_minus">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_plus">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_equals">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 255, 0);</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_exp</sender>
   <signal>inputRejected()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
